python/PyQt6/core/__init__.py.in

```diff
@@ -515,12 +515,30 @@ try:
 
    QgsRasterBlock.as_numpy = _raster_block_as_numpy
 
+   def _raster_layer_as_numpy(self, use_masking=True, bands: _typing.Optional[_typing.List[int]] = None):
```

Comment:
Is it possible to add the type hint for the return type here?

Reply:
I guess it should be ` -> _numpy.ndarray`?

Reply:
Actually `_typing.List[_typing.Union[_numpy.ndarray, _numpy.ma.MaskedArray]]`